CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## Version 2.1.1
+
+- [Address the issue where DependencyCollection breaks Azure Storage Emulator calls](https://github.com/Microsoft/ApplicationInsights-aspnetcore/issues/488)
+- [Support setting request operation name based on executing Razor Page](https://github.com/Microsoft/ApplicationInsights-aspnetcore/issues/430)
+- [Fixed ITelemetryProcessor dependency injection failure when using 3rd party IoC Container](https://github.com/Microsoft/ApplicationInsights-aspnetcore/issues/482)
+- Updated SDK version dependency to 2.4.1 for DependencyCollector.
+
```

Comment:
Please check the history and add in at least the following items (but with better descriptions than I provided here):
- Updated the ILogger logging of exceptions to send ExceptionTelemetry instead of TraceTelemetry.
- Added sanitization of authenticated user name to the JavaScript snippet output.
- Anything else that changed in develop since 2.1.0.